### PR TITLE
Add section for configuring custom menu items in Astro

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/3653

I thought it would make more sense to put all of this info in the Learn guide, rather than have an Astro-specific section that only explains what code to add.

I'll link out to here from the release note after we publish

### CODE REVIEW

File: learn/using-airflow-plugins.md [hunk 1]
```diff
@@ -111,12 +118,49 @@ class MyMenuItemsPlugin(AirflowPlugin):
...
+        "href":  str(baseUrl) + "/testappbuilderbaseview",
+    }
+    ```
+
+    You can then specify the menu item, in this example `my_new_endpoint` in `appbuilder_menu_items`.
+
+```
 
-Both additional menu items are added to the `app_builder_menu_items` component of a plugin called `Menu items plugin` which is defined in the `MyMenuItemsPlugin` class. The screenshot below shows the additional menu items in the Airflow UI.
+The screenshot below shows the additional menu items in the Airflow UI.
```
Comment (chomipi88) [inline learn/using-airflow-plugins.md:163]: This part is not rendering properly

File: learn/using-airflow-plugins.md [hunk 2]
```diff
@@ -111,12 +118,49 @@ class MyMenuItemsPlugin(AirflowPlugin):
...
+    my_new_endpoint = { 
+        "name": "Custom Menu",
+        "label": "DAGs Folder",
+        "href":  str(baseUrl) + "/testappbuilderbaseview",
+    }
+    ```
+
+    You can then specify the menu item, in this example `my_new_endpoint` in `appbuilder_menu_items`.
+
+```
```
Comment (jwitz) [inline learn/using-airflow-plugins.md:161]:
````suggestion
```
:::
````

**Comment by jwitz** [comment]
@chomipi88 Permissions changes are in this PR: https://github.com/astronomer/docs/pull/3668

**Review by lzdanski — APPROVED** [review]
Content changes LGTM, but might need a review from @kentdanas or @TJaniF since this is a learn guide.

**Review by chomipi88 — APPROVED** [review]

**Comment by jwitz** [comment]
Sorry @kentdanas and @TJaniF Getting this merged so that we can publish release notes, but please comment anything else you want to see in this section and I'll update it ASAP!

**Comment by TJaniF** [comment]
@jwitz thank you! Sorry didnt realize this had a deadline, will 👀 this evening :)

**Comment by TJaniF** [comment]
@jwitz Made a PR #3678 :)

**Pull request merged**